packages/flutter/lib/src/utils/parse_live_list.dart

```diff
@@ -272,8 +271,6 @@ class _ParseLiveListElementWidgetState<T extends sdk.ParseObject>
       sizeFactor: widget.sizeFactor,
       child: AnimatedSize(
         duration: widget.duration,
-        // ignore: deprecated_member_use
-        vsync: this,
```

Comment:
Is there no alternative introduced for vsync?

Reply:
According to [Flutter's commit](packages/flutter/lib/src/widgets/animated_size.dart), this property is marked as deprecated and its value is ignored.  So there is no need to introduce other alternatives.  Thanks!

Reply:
I get it, thank you